chapter_natural-language-processing/sentiment-analysis-cnn.md

```diff
@@ -304,16 +306,16 @@ loss = gloss.SoftmaxCrossEntropyLoss()
 
 在实验中，我们使用准确率作为评价模型的指标。
 
-```{.python .input  n=12}
+```{.python .input  n=95}
 def eval_model(features, labels):
```

Comment:
Let's use gb.evaluate_accuracy(). This function will be invoked by gb.train. Thus, just use gb.train() like in the following example:
http://zh.gluon.ai/chapter_computer-vision/fine-tuning.html

The definitions of gb.train() is available at:
https://github.com/mli/gluon-tutorials-zh/blob/master/gluonbook/utils.py#L371

If it's impossible to simply invoke gb.train(), just change the function name to evaluate_accuracy() and make it be able to evaluate on multi-gpu, example:
https://github.com/mli/gluon-tutorials-zh/blob/master/gluonbook/utils.py#L106